hathor/simulator/simulator.py

```diff
@@ -178,6 +178,7 @@ def create_artifacts(self, builder: Optional[Builder] = None) -> BuildArtifacts:
             .build()
 
         artifacts.manager.start()
+        self._clock.running = True
```

Comment:
@msbrogli this will probably be a relevant line to change in https://github.com/HathorNetwork/hathor-core/pull/584

Reply:
Referenced in https://github.com/HathorNetwork/hathor-core/pull/584#issuecomment-1632904296